client/utils/loadComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+const loadComponent = (importCall) => {
+  return class extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        component: () => <div>Loading...</div>
```

Comment:
Add a semicolon here.

Reply:
Also kindly add this comment on top of this line.

```TODO: Custom Spinner component to be implemented for use here.```

Reply:
@BolajiOlajide Alright...Thanks.